.github/workflows/pull-request-test.yml

```
@@ -8,14 +8,11 @@ on:
       - closed
       - edited
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
```

```suggestion
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```
maybe? so it cancels everything on the same PR if it runs on a PR?

---

ref is the branch, no?

---

hmm I guess it should. But I see a lot of repos using the above in the wild.

---

I think if you had multiple PRs to merge the same branch into different branches (like a hotfix branch into master and develop if you follow proper git-flow) and the trigger to run actions is on PR, one PR would cancel the other PR?

---

gitflow is beyond my ability of comprehension. If we have the PR number though do we need the ref in there at all? This workflow only triggers on PRs